spec/fields/access_condition_spec.rb

```diff
@@ -84,5 +85,13 @@ def mods_display_access_condition(mods_record)
         expect(fields.first.values.first).not_to include('<a.*>')
       end
     end
+
+    describe 'use and reproduction' do
+      it 'preserves newlines' do
+        fields = mods_display_access_condition(@custom_use_reproduction).fields
+        expect(fields.length).to eq(1)
+        expect(fields.first.values).to eq ["Special use agreement\nText of the agreement"]
```

Comment:
If I understand the intended display correctly, we will want a blank line in between the special and default use statements (`/n/n`). So, use two linebreaks here and in the fixture?

Reply:
Oh, good point! Thank you